internal/controllers/nmc_reconciler.go

```diff
@@ -78,13 +58,14 @@ func NewNMCReconciler(
 	workerCfg *config.Worker,
 	recorder record.EventRecorder,
 	nodeAPI node.Node,
+	kmodLoader pod.KmodLoader,
```

Comment:
lets keep podManager variable name, much more clearer

Reply:
I will use `WorkerPodManager` - as you suggested. `podManager` is too generic and we have another `podManager` interface coming in this package in a following PR - I try to give them meaningful names.

Reply:
Fixed.